man/fi_endpoint.3.md

```diff
@@ -616,19 +616,20 @@ if data ordering is not maintained, message ordering may be.
 : Read after write size.  If set, an RMA or atomic read operation
   issued after an RMA or atomic write operation, both of which are
   smaller than the size, will be ordered.  The RMA or atomic read
-  operation will see the results of the previous RMA or atomic write.
+  operation will see the results of the previous RMA or atomic write
+  at the target memory location.
```

Comment:
I'm not sure this is clearer.  The phrase 'at the target memory location' could apply to the write, the result, or the read operation.  I think the original text is better here.